docs/how-to/express-migration.md

```diff
@@ -31,33 +31,88 @@ When using Encore.ts you get a lot of built-in features without having to instal
 | [Request validation](/blog/event-loops)                     |           [Cron Jobs](/docs/ts/primitives/cron-jobs)           |             [Local tracing](/docs/observability/tracing) |
 | [Local Development Dashboard](/docs/observability/dev-dash) |
 
-## App architecture
-
-Like Express, Encore.ts in not opinionated in how you structure your code.
-It is recommended to use one Encore app for your entire backend application, as this lets Encore
-build an application model that spans your entire system. This is necessary to get the most value out of many features like [distributed tracing](https://encore.dev/docs/observability/tracing) and [Encore Flow](https://encore.dev/docs/observability/encore-flow).
-Learn more in our [App Structure docs](https://encore.dev/docs/ts/develop/app-structure).
+## Migration guide
 
-### Monolith or Microservices
+We have outlined two different strategies you can use to migrate your existing Express.js application to Encore.ts. Pick the strategy that suites your situation and application the the best.
 
-Encore is not opinionated about monoliths vs. microservices. It does however let you build microservices applications with a monolith-style developer experience. For example, you automatically get IDE auto-complete when making [API calls between services](https://encore.dev/docs/ts/primitives/services-and-apis#calling-apis), along with cross-service type-safety.
+<Accordion>
 
-## Migration guide
+### Forklift migration (quick start)
 
 You can use a forklift migration strategy and move the entire application over to Encore.ts in one shot by wrapping your existing HTTP router in a catch-all handler.
 
-This can be relatively straightforward if your existing system is a monolith, or smaller distributed system, and does not rely on many unsupported [cloud primitives](/docs/primitives/overview).
+**Approach benefits**
+
+- You can get your application up and running with Encore.ts quickly and start moving features over to Encore.ts while the rest of the application is still untouched.
+- You will see a performance boost right away because the HTTP layer is now running on the Encore Rust runtime. But to get the full performance benefits, you will need to start using Encore's [API declarations](/docs/ts/primitives/services-and-apis) and [infrastructure declarations](/docs/ts#explore-how-to-use-each-backend-primitive).
+
```

Comment:
```suggestion
- You will see a partial performance boost right away because the HTTP layer is now running on the Encore Rust runtime. But to get the full performance benefits, you will need to start using Encore's [API declarations](/docs/ts/primitives/services-and-apis) and [infrastructure declarations](/docs/ts#explore-how-to-use-each-backend-primitive).

```